aws_lambda_powertools/logging/logger.py

```diff
@@ -583,6 +583,12 @@ def debug(
     def append_keys(self, **additional_keys: object) -> None:
         self.registered_formatter.append_keys(**additional_keys)
 
+    def get_current_keys(self) -> Dict[str, Any]:
+        if hasattr(self.registered_formatter, "log_format"):
```

Comment:
can we check for instance type instead of doing "introspection"?

```suggestion
        if isinstance(self.registered_formatter, LambdaPowertoolsFormatter)::
```